### DISCUSSION TITLE
Allow headless extended statistics calculation

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#14469] [Ideas] [answered]
This is only a minor pain point, but on occasion I want to look at the extended stats for one of my plays (i.e. hit error graphs, unstable rate, etc). Currently, to do that, I need to open the reply, skip to the end and then wait while the seek happens (which appears to be capped at a certain speed, so it can take an annoyingly long time even on a good computer). It would be great if either that seek speed could be uncapped or if there was an option to recalculate the stats without watching the gameplay, which would presumably be more performant and fewer clicks.

**Comment by peppy** [comment, marked as answer]
yes there will eventually be a better solution here. either cached or automatic.

**Reply by memesnightmares** [reply]
Cool, how does that sort of feature get tracked in this project? Should I close this discussion? I'm not sure how GH discussions work :P

**Reply by peppy** [reply]
It's fine to leave this open (discussions actually can't be closed). I'll open a tracking issue as this *is* something we need fixed/implemented.

**Reply by frenzibyte** [reply]
Tracking in https://github.com/ppy/osu/issues/14472 for reference.
